buffer/src/main/java/io/netty/buffer/ByteBufUtil.java

```diff
@@ -388,6 +388,30 @@ public static long swapLong(long value) {
     }
 
     /**
+     * Writes a big-endian 16-bit short integer to the buffer.
+     */
+    @SuppressWarnings("deprecation")
+    public static ByteBuf writeShortBE(ByteBuf buf, int shortValue) {
+        return buf.order() == ByteOrder.BIG_ENDIAN? buf.writeShort(shortValue) : buf.writeShortLE(shortValue);
```

Comment:
@normanmaurer - is this @trustin's preference? do you care if we change the style? I feel like this has come up before ... seems strange to not have spaces separating tokens.